src/core/language.ts

```diff
@@ -31,6 +31,7 @@ export async function readLanguageTranslations(
     if (existsSync(file)) {
       files.push(Deno.realPathSync(file));
       const errMsg = "Validation of format language object failed.";
+      const formatLanguageSchema = getSchemaDefinition("format-language");
```

Comment:
We should make sure that this schema is open, as users may create language files that have new keys created for later versions that they nevertheless want to use with an older version.

Reply:
Ok. FWIW, it is. We denote a closed schema in yaml by 

```
object:
  ...
  additionalProperties: false
```